### PR TITLE
[CFM-454] Send GO/GA notification on new member that has joined, open group.

### PR DESCRIPTION
**Pull request by anvmn**
## This PR is dependent on #1059, #1061

https://issuetracker.amplexor.com/jira/browse/CFM-454

### CODE REVIEW

**Comment by anvmn** [comment]
selection_115

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 1]
```diff
@@ -41,6 +41,31 @@ function c4m_user_notifications_moderated_group_notify_admins_membership_accepte
...
+    // Return early, if message was not created properly.
+    return;
+  }
+  c4m_message_load_tokens__c4m_new_member_joined($message);
+
+  $wrapper = entity_metadata_wrapper('message', $message);
+  $wrapper->field_group_node->set($membership->gid);
+  $wrapper->field_new_member->set($membership->etid);
+  $wrapper->field_group_membership->set($membership->id);
+  message_notify_send_message($wrapper->value());
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: `message_subscribe_send_message()`

**Comment by anvmn** [comment]
This PR is dependent on https://github.com/capacity4dev/capacity4more/pull/1061, where message field_related_user and field_group_membership are introduced.

**Comment by ordavidil** [comment]
re-roll

File: config/bin/src/dummy.sh [hunk 2]
```diff
@@ -22,5 +22,7 @@ function dummy_content_modules {
 # 'dummy_content_migration' is set to 1 during the process, 0 when done.
 ##
 function dummy_content_migrate {
+  drupal_drush vset --exact dummy_content_migration 1
```
Comment (ordavidil) [inline config/bin/src/dummy.sh]: @anvmn 
why do we need this one again ?

Comment (anvmn) [inline config/bin/src/dummy.sh]: @ordavidil 
I'll see about it...

**Comment by anvmn** [comment]
## Replaced by #1189

**Pull request closed without merging**